mailpoet/tests/acceptance/Settings/CreateNewWordPressUserCest.php

```diff
@@ -42,7 +42,7 @@ public function sendConfirmationEmail(\AcceptanceTester $i) {
     $i->checkEmailWasReceived($emailTitle);
     $i->click(Locator::contains('span.subject', $emailTitle));
     $i->switchToIframe('#preview-html');
-    $i->click('Click here to confirm your subscription.');
+    $i->click('Click here to confirm your subscription');
```

Comment:
Maybe I'm missing something obvious here, but it is not immediately clear to me why this change was needed. For cases like this, I would suggest including in the commit message description an explanation of why removing the dot in this file and in SubscribeOnRegistrationPageCest.php was necessary and what exactly was broken that was fixed.